docs/Radio-hardware-specifications-for-EdgeTX.md

```diff
@@ -30,9 +35,19 @@
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <img src="https://c.a.segger.com/fileadmin/images/products/J-Trace/jtrace-pinout.png">
 
 * USB-C device mode connection
+    * H7R radios must support Hi-Speed (480 MBit/s) USB
 * For radios with an internal module, lines to put the module into flashing mode to perform pass-through flashing
 * Unused GPIO pins shall be connected with a 10k resistor either to GND or VCC. The initial hw revision of a radio shall have them tied to GND, further revision shall use one or multiple of those pins to mark different hw revision of the radio to allow auto detection of the radio version or variant.
 
+### hardware design guidlines
+* all high speed interfaces like Quad/Octo/Hexa SPI, SDRAM, USB and SDIO shall be length / time of flight equalised
+  * relative timing constrains, e.g. for different SDRAM wires, shall be respected
+* STM32H7 CPUs shall be used with Quad or Octo SPI flash that supports DDR and at least 100 MHz of DDR SPI clock speed
+* for connecting the Quad/Octp SPI flash and RAM the pins used on the evaluation boards shall be used
```

Comment:
* for connecting the Quad/Octo SPI flash and RAM, the same I/O pins should be used as on the following ST evaluation boards:

Reply:
Maybe a good idea to reference the user manual for these boards and page number the schematic is on? i.e. a random STM32 board, figure 69, page 101, https://www.st.com/content/ccc/resource/technical/document/user_manual/group0/48/5f/49/8e/5d/44/47/e3/DM00385071/files/DM00385071.pdf/jcr:content/translations/en.DM00385071.pdf